tests/test_e2e.py

```diff
@@ -0,0 +1,47 @@
+import os
+import asyncio
+from playwright.async_api import async_playwright
+
+'''
+To setup on docker container:
+pip install playwright pytest-playwright asyncio
+playwright install
+apt-get install -y gstreamer1.0-libav libnss3-tools libatk-bridge2.0-0 libcups2-dev libxkbcommon-x11-0 libxcomposite-dev libxrandr2 libgbm-dev libgtk-3-0
+
+To setup locally:
+pip install playwright pytest-playwright asyncio
+pip install -e .
+playwright install --with-deps
+
+To run test locally in debug mode:
+PWDEBUG=1 pytest -s
+'''
```

Comment:
Can some of this bit be added to our Dockerfile?
Also, perhaps you could add the "setup locally" bit to our files in `/documentation` ?